joomla/joomla-cms
# [4.0] Fix nested items reorder

Pull Request for Issue #34855.

### Summary of Changes
This PR fixes re-order nested items (categories/menu items) as described in https://github.com/joomla/joomla-cms/issues/34855. I did two things here:

- Move the code to save order back to `drop` instead of `dragend`. I haven't had time to check to see why it is not working for `dragend` but we always use `drop` before and it works, so it makes sense to make the change
- Remove the code to initialize data-order for each item and reset it on dragend. We do not use data-order in our script at the moment, so no need to set data for it.

### Testing Instructions
1. Confirm the issue https://github.com/joomla/joomla-cms/issues/34855
2. Update your site to update package for this PR
3. Try to change ordering of categories and make sure it is working as expected. It would be good if you can test change ordering of menu items, modules, articles and make sure it is still working, too.
